platforms/platform-alexa/src/output/templates/AudioPlayerPlayOutput.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+import { AudioItem, PlayBehavior, PlayBehaviorLike } from '../models';
+import _merge from 'lodash.merge';
+
+export interface AudioPlayerPlayOutputOptions extends OutputOptions {
```

Comment:
I think we should add `metadata` as well: https://developer.amazon.com/en-US/docs/alexa/custom-skills/audioplayer-interface-reference.html#directives

Reply:
`metadata` is part of `audioItem`